micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/ServiceRestClientConfigurationSupport.groovy

```diff
@@ -51,6 +50,8 @@ class ServiceRestClientConfigurationSupport {
     @Value('${rest.client.maxLogResponseChars:4096}')
     int maxLogResponseChars
 
+    @Value('${rest.client.responseErrorMessageLogging:ERROR}')
+    String responseErrorMessageLogging
```

Comment:
right, but if I just put enum type here, incorrect value results in context startup failure (which is quite harsh consequence for something so insignificant as logging configuration) - unless I create custom converter that defaults on incorrect value, register it in spring context and in conversion service, etc